src/NmeaParser.Shared/Nmea/AltitudeUnit.cs

```diff
@@ -0,0 +1,40 @@
+﻿//
+// Copyright (c) 2016 Morten Nielsen
+//
+// Contributors:
+// Stephen Kennedy, Copyright (c) 2016 Gloucester Software Ltd.
+//
+// Licensed under the Microsoft Public License (Ms-PL) (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://opensource.org/licenses/Ms-PL.html
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+namespace NmeaParser.Nmea
+{
+    /// <summary>
+    /// Altitude unit
+    /// </summary>
+    public enum AltitudeUnit
+    {
+        /// <summary>
+        /// Unknown
+        /// </summary>
+        Unknown,
+        /// <summary>
+        /// Feet
+        /// </summary>
+        Feet,
```

Comment:
Feet should probably be more specific for a generic unit (there are many kinds).